src/main/java/org/libra/Constants.java

```diff
@@ -7,11 +7,63 @@
 import org.libra.types.ChainId;
 import org.libra.utils.AccountAddressUtils;
 
+/**
+ * Defines constants for static data and enum type values.
+ *
+ * See the following Libra JSON-RPC response type documents for more details of enum type values:
+ * <a href="https://github.com/libra/libra/blob/master/json-rpc/docs/type_account.md#type-account">Account</a>
+ * <a href="https://github.com/libra/libra/blob/master/json-rpc/docs/type_event.md#event-data">EventData</a>
+ * <a href="https://github.com/libra/libra/blob/master/json-rpc/docs/type_transaction.md#type-vmstatus">VmStatus</a>
+ * <a href="https://github.com/libra/libra/blob/master/json-rpc/docs/type_transaction.md#type-transactiondata">TransactionData</a>
+ */
 public interface Constants {
+    // static account addresses
     AccountAddress ROOT_ACCOUNT_ADDRESS = AccountAddressUtils.create("0000000000000000000000000A550C18");
     AccountAddress CORE_CODE_ADDRESS = AccountAddressUtils.create("00000000000000000000000000000001");
 
+    // ChainIDs, Testnet id see Testnet#CHAIN_ID
     ChainId MAINNET_CHAIN_ID = new ChainId((byte) 1);
     ChainId DEVNET_CHAIN_ID = new ChainId((byte) 3);
     ChainId TESTING_CHAIN_ID = new ChainId((byte) 4);
+
+    // AccountRole#type field values
+    String ACCOUNT_ROLE_UNKNOWN = "unknown";
```

Comment:
wouldn't it be better if the string value is same as constant? Like ACCOUNT_ROLE_UNKNOWN = "account_role_unknown", it'd be more info too